src/main/java/graphql/normalized/ExecutableNormalizedField.java

```diff
@@ -61,20 +65,108 @@ private ExecutableNormalizedField(Builder builder) {
         this.parent = builder.parent;
     }
 
+    @Deprecated // doesn't really work
```

Comment:
See the doco below for why this is wrong.

Reply:
I would suggest removing it then ... this is not a public api and we don't need to deprecate thing we don't want